docs/endpoints/multi_user.rst

```diff
@@ -456,6 +463,57 @@ a custom ``globus-compute-endpoint`` wrapper:
 
 (The use of ``exec`` is not critical, but keeps the process tree tidy.)
 
+
+Configuring to Accept Multiple Python Versions
+==============================================
+
+By default, Globus Compute serializes task submissions via `dill`_ with a method that
+uses Python bytecode. That's to say it does *not* serialize the source code unless asked
+to, for both technical and historical reasons, but this can sometimes cause issues.
+Because the underlying representations that Python uses for bytecode are subject to
+change at the whim of the Python developers, if the Python version running the **SDK**
+that's used to serialize and submit a task is different from the Python version of the
+**worker** that deserializes and runs the task, the worker can error (in ways that are
+often hard to debug because they happen at a low level in Python).
+
+As a result, our recommendation is to always keep Python versions in sync between SDK
+invocations and endpoint workers. However, this can be limiting for certain workflows
+where cross-version compatibility is desirable. This can sometimes be alleviated with
```

Comment:
"this can be" is passive voice.  Not wrong, but suggest rewording to a more active statement.